### ISSUE TITLE
iCubGenova02 branches

### ISSUE THREAD

**Issue by fiorisi** [robotology/robots-configuration#79]
For iCubGenova02 currently we have the following branches:
- iCubGenova02-04/fixGyroscopesFullscale
- iCubGenova02_V2-5_plus_soles_speed
- iCubGenova02_V2-5_plus_soles
- iCubGenova02_noRightFTSensor
- iCubGenova02/createConfigInertialsMultipleAnalogSensors

Now that the [strain2 FT have been installed](https://github.com/robotology/icub-support/issues/690) probably we can clean up a bit.
Proposed approach:
- DELETE iCubGenova02_noRightFTSensor
- DELETE iCubGenova02/createConfigInertialsMultipleAnalogSensors
- MERGE devel into iCubGenova02_V2-5_plus_soles
- MERGE devel into iCubGenova02_V2-5_plus_soles_speed  (help needed by @ale-git) 

if possible:
- MERGE iCubGenova02_V2-5_plus_soles_speed into iCubGenova02_V2-5_plus_soles (help needed by @ale-git)
- DELETE iCubGenova02-04/fixGyroscopesFullscale

Once the new soles (15 [deg] offset) have been fully tested, we could merge into devel.

**Comment by traversaro** [comment]
Ok!

**Comment by nunoguedelha** [comment]
@fiorisi , you can delete `iCubGenova02-04/fixGyroscopesFullscale`, that change is already merged into the master branch.

Please **DON'T** delete `iCubGenova02/createConfigInertialsMultipleAnalogSensors`, I will handle it as soon as I arrive at IIT at 2pm.

**Comment by nunoguedelha** [comment]
@fiorisi , the branch `iCubGenova02/createConfigInertialsMultipleAnalogSensors` has been PRed (https://github.com/robotology/robots-configuration/pull/71). I'll let you know as soon as it's merged so that we clean it up from the repos.

**Comment by fiorisi** [comment]
Thanks to the work of @nunoguedelha the branches have been cleaned up.
Currently the robot iCubGenova02 is fully working in the `devel` branch and the only two additional branches are (other 2 should have dedicated pull requests):
- `iCubGenova02_V2-5_plus_soles_speed`: used for jump test in velocity
- `iCubGenova02_V2-5_plus_soles`: used for jump tests

Currently, these branches can not be merged in `devel` because the current and PWM limits have been increased to the maximum values (e.g. see the [left leg file](https://github.com/robotology/robots-configuration/blob/iCubGenova02_V2-5_plus_soles/iCubGenova02/hardware/motorControl/left_leg-eb7-j4_5-mc.xml)). Nevertheless, if this experimental configuration is stable we might consider merging.

**NOTE**:
We noticed that all the [low level configuration files](https://github.com/robotology/robots-configuration/tree/devel/iCubGenova02/hardware/motorControl) are still using the `machine_units` settings (instead of the new `dutycycle_percent`). We decided to don't merge the modified files from the `iCubGenova02_V2-5_plus_soles` branch because of two reasons:
- the files are incomplete
- the parameters are not tuned

We can schedule a day to correctly estimate the parameters and update the low level configuration files.

cc @randaz81 @FabioBergonti @hu-yue
